error CS1525: Invalid expression term '>'

I have this compile error with the Release [4.0.0-rc.2](https://github.com/RevenueCat/purchases-unity/releases/tag/4.0.0-rc.2)

`Assets/RevenueCat/Example/PurchasesListener.cs(340,80): error CS1525: Invalid expression term '>' `

I'am using **Unity 2020.3.8f1** on **Macbook pro m1** with Rider.

![Screenshot 2022-08-02 at 12 41 29](https://user-images.githubusercontent.com/6715157/182355934-9ff37a5e-140f-4644-8c76-3de6e72ee7e0.png)

--- a/RevenueCat/Example/PurchasesListener.cs
+++ b/RevenueCat/Example/PurchasesListener.cs
@@ -46,10 +46,16 @@ public class PurchasesListener : Purchases.UpdatedCustomerInfoListener
         CreateButton("Toggle simulatesAskToBuyInSandbox", ToggleSimulatesAskToBuyInSandbox);
         CreateButton("Is Anonymous", IsAnonymous);
         CreateButton("Get AppUserId", GetAppUserId);
+        CreatePurchasePackageButtons();
 
         var purchases = GetComponent<Purchases>();
         purchases.SetDebugLogsEnabled(true);
         purchases.EnableAdServicesAttributionTokenCollection();
+    }
+
+    private void CreatePurchasePackageButtons()
+    {
+        var purchases = GetComponent<Purchases>();
         purchases.GetOfferings((offerings, error) =>
         {
             if (error != null)
@@ -59,15 +65,13 @@ public class PurchasesListener : Purchases.UpdatedCustomerInfoListener
             else
             {
                 Debug.Log("offerings received " + offerings.ToString());
-                var yOffset = 0;
 
                 foreach (var package in offerings.Current.AvailablePackages)
                 {
                     Debug.Log("Package " + package);
                     if (package == null) continue;
                     var label = package.PackageType + " " + package.StoreProduct.PriceString;
-                    CreateButton(label, () => ButtonClicked(package));
-                    yOffset += yPaddingForButtons;
+                    CreateButton(label, () => PurchasePackageButtonClicked(package));
                 }
             }
         });
@@ -124,7 +128,7 @@ public class PurchasesListener : Purchases.UpdatedCustomerInfoListener
     }
 
 
-    private void ButtonClicked(Purchases.Package package)
+    private void PurchasePackageButtonClicked(Purchases.Package package)
     {
         var purchases = GetComponent<Purchases>();
         purchases.PurchasePackage(package, (productIdentifier, customerInfo, userCancelled, error) =>

--- a/RevenueCat/Example/PurchasesListener.cs
+++ b/RevenueCat/Example/PurchasesListener.cs
@@ -338,7 +338,7 @@ public class PurchasesListener : Purchases.UpdatedCustomerInfoListener
             {
                 var products = GetPackages(offerings)
                     .Select(package => package.StoreProduct) // map to product ids
-                    .Where(storeProduct => storeProduct.Discounts is { Length: > 1 }) // map to product ids
+                    .Where(storeProduct => storeProduct.Discounts != null && storeProduct.Discounts.Any()) // map to product ids
                     .ToArray();
 
                 if (products.Length == 0)

